graph/types/schema.graphqls

```diff
@@ -62,7 +62,7 @@ input DiscussionInput {
 type Mutation {
   addDiscussionParticipant(discussionID: String!, userID: String!, discussionParticipantInput: AddDiscussionParticipantInput!): Participant!
   addPost(discussionID: ID!, participantID: ID!, postContent: PostContentInput!): Post!
-  postImportedContent(discussionID: ID!, participantID: ID!, contentID: ID!): Post! # TODO: Need clarity on UX for this. Keeping it simple for now
```

Comment:
updated here. let me know if this is implemented in the frontend/is a breaking change. didn't see anything in the app repo so should be fine

Reply:
Should be good.

Sorry for ignorance though -- is there a situation where the mod posts directly? I think based on Chris's mocks we want it to come from the moderator participant in that case.

Reply:
where can I track down these mocks? I can revert this easily

Reply:
updated